packages/email/email.d.ts

```diff
@@ -11,7 +11,7 @@ export namespace Email {
   interface ExtraMailOptions {}
   type EmailOptions = { mailComposer: MailComposer } | (ExtraMailOptions & SendMailOptions)
 
-  interface CustomEmailOptions extends EmailOptions {
+  type CustomEmailOptions = EmailOptions & {
```

Comment:
I thought we've decided to leave it as an interface?

Reply:
No, we added the `ExtraMailOptions` interface so that it could be extended. But as currently written, this throws an error:

```
.meteor/local/types/node_modules/package-types/email/package/os/packages/email/email.d.ts:14:40 - error TS2312: An interface can only extend an object type or intersection of object types with statically known members.

14   interface CustomEmailOptions extends EmailOptions {
                                          ~~~~~~~~~~~~
```

In the `DefinitelyTyped` version, we switched this to a type instead of an interface for that reason, but I missed it when updating this declaration from my original patch.

Reply:
Ah, OK, good.